### ISSUE TITLE
Warnings on test run

### ISSUE THREAD

**Issue by TZubiri** [python-bonobo/bonobo#231]
tests/test_commands.py::test_run_module[runner1]
  /opt/python/3.6.3/lib/python3.6/runpy.py:125: RuntimeWarning: 'bonobo.examples.types.strings' found in sys.modules after import of package 'bonobo.examples.types', but prior to execution of 'bonobo.examples.types.strings'; this may result in unpredictable behaviour
    warn(RuntimeWarning(msg))

Maybe we should ignore or fix these warnings? I have no clue what they mean, but I had to go through travis logs to ensure it wasn't an artifact of my environment.

**Comment by hartym** [comment]
Yes, those warnings are annoying. I never silenced them because even if it sounds normal, related to what happened, I'm not really sure that this side effect of runpy is something we want, so kept it as a reminder.

For sure it's not bad in 99% of situations, but I'm considering a backup/restore of sys.modules.
